File: src/main/java/com/tom/guess/Game.java
package com.tom.guess;

import java.sql.SQLOutput;
import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10)+1;//(0~9)+1
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (secret !=number){
            System.out.print("Please enter a number: ");
            number = scanner.nextInt();
            if (number < secret){
                System.out.println("Higher");
            }else if(number> secret){
                System.out.println("Lower");
            }else {
                System.out.println("Great,the number is: " + number);
            }
        }


        /*int i = 0;
        do{
            System.out.println(i);
            i++;//i= i +1;
        }while (i < 3);*/
    }
}

File: src/main/kotlin/com/tom/Game.kt
package com.tom

import java.util.*

fun main() {
    val secret = Random().nextInt(10)+1
    println(secret)
//    val scanner = Scanner(System.`in`)
    var number = 0
    while (number != secret){
        print("Please enter a number: ")
//        number = scanner.nextInt();
        number = readLine()!!.toInt()
        if (number > secret){
            println("Lower")
        }else if (number < secret){
            println("Higher")
        }else{
            println("Great,the number is $number")
        }
    }

}